pynuodb/encodedsession.py

```diff
@@ -438,30 +438,27 @@ def fetch_result_set(self, statement):
         handle = self.getInt()
         colcount = self.getInt()
 
-        col_num_iter = range(colcount)
-        for _ in col_num_iter:
+        # skip the header labels
+        for _ in range(colcount):
```

Comment:
I'm not sure why this is done, but anyway I didn't change it.

Reply:
where is the change coming from then? Does it work?

Reply:
got it, it sounded like you did not know where the change came from :)